From 7a49f8e3650580613e2ca6f5ab8b7b08e6685966 Mon Sep 17 00:00:00 2001
From: Dinesh Karthikeyan <quic_dinek@quicinc.com>
Date: Wed, 24 May 2023 20:13:32 +0530
Subject: [PATCH] cfg80211: Increase grace period for PRE CAC timeout

Increase the grace period for PRE CAC timeout to 4 seconds to bring up
5G vap in DFS channel to handle MAX number of MLDs case.

Signed-off-by: Dinesh Karthikeyan <quic_dinek@quicinc.com>
---
 net/wireless/reg.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: backports-20220822-5.4.213-ef7197996efe/net/wireless/reg.h
===================================================================
--- backports-20220822-5.4.213-ef7197996efe.orig/net/wireless/reg.h	2023-05-30 12:44:20.539353585 +0530
+++ backports-20220822-5.4.213-ef7197996efe/net/wireless/reg.h	2023-05-30 12:44:38.699413210 +0530
@@ -152,7 +152,7 @@ bool regulatory_indoor_allowed(void);
  * value is used for Non-ETSI domain.
  * TODO: May be make this timeout available through regdb?
  */
-#define REG_PRE_CAC_EXPIRY_GRACE_MS 2000
+#define REG_PRE_CAC_EXPIRY_GRACE_MS 4000
 
 #define REG_BG_PRE_CAC_EXPIRY_GRACE_MS 2000
 
